File: firstLast6/FirstLast6.java
package firstLast6;

public class FirstLast6 {
    //Retorne true se há 6 dentro do Array como primeiro elemnto ou último, o Array deverá ter um "length" maior que 1.

    public boolean firstLast6(int[] nums) {
        return (nums[0] == 6 || nums[nums.length-1] == 6);
    }
}

File: Python/List-1/first_last6.py
"""
Dada uma lista numérica "nums", retorne true se o seu primeiro ou último elemento
forem um "6". A lista terá um tamanho de pelo menos 1.
"""


def first_last6(nums):
    return nums[0] == 6 or nums[-1] == 6


print(first_last6([1, 2, 3]))
